doc/usecases/core/UC_key_create.md

```diff
@@ -0,0 +1,25 @@
+# Use Case: Create `Key`
+
+## Summary
+
+- **Scope:** `libelektra-core`
+- **Level:** Developer Goal
+- **Actors:** Core, Caller
+- **Brief:** Caller creates new `Key` instance
+
+## Scenarios
+
+- **Precondition:** -
```

Comment:
I will describe the error scenario. Although IMO this is a limitation of the current API and not something imposed by the use case. That's why I didn't mention it.

If the API looked like this:

```c
Key * keyNew (elektraNamespace namespace, const char * unescapedName, size_t nameSize);
```

Then there cannot be an invalid name.

Reply:
I ended up adding it as a precondition after all. I intentionally just said "must be valid" without saying what "valid" means. That way it still covers all cases, because it may just be that all names are "valid".